### PR TITLE
Release

### PR DESCRIPTION
**Pull request by windkomo**
- Label cusotmization
- appId => botId
- Default images are included into script, custom ones are not
- Remove server stuff, add webpack-dev-server and prepublish hook

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+## Install
+
+<script> tag using CDNs
+
+jsDelivr is a global CDN delivery for JavaScript libraries.
+
+To include the latest release:
+
+<script src="https://cdn.jsdelivr.net/npm/botfuel-webchat-client"></script>
+
```
Comment (kevinadda) [inline README.md]: Can you already document for older releases ?

Comment (windkomo) [reply README.md]: done 👍

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/default.min.css">
+    <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+    <title>Botfuel WebChat Demo</title>
```
Comment (kevinadda) [inline src/index.html]: should it be demo?

File: webpack.config.js [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
 const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
-  entry: './src/App.jsx',
+  entry: [
+    // 'react-hot-loader/patch',
+    // `webpack-dev-server/client?http://localhost:${process.env.PORT || 7000}`,
+    // 'webpack/hot/only-dev-server',
+    // 'babel-regenerator-runtime',
```
Comment (kevinadda) [inline webpack.config.js]: should we keep these?

Comment (windkomo) [reply webpack.config.js]: Maybe as a reminder to setup hot reload

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    <div id="botfuel"></div>
+</body>
+<script>
+    BotfuelWebChat.init({
+        applicationId: '1409615561904',
+        theme: {
+            colors: {
+                main: '#3d3d3d',
+                mainLight: '#f5f5f5',
+                primary: '#f4b600',
```
Comment (kevinadda) [inline src/index.html]: we'll need to change these before the end of the sprint, I guess it won't look good on the page

**Pull request merged**